libparsec/crates/protocol/schema/authenticated_cmds/vlob_fetch.json5.new

```diff
@@ -0,0 +1,81 @@
+[
```

Comment:
What is this extension `.json5.new` ?

Reply:
just a file I shouldn't have commited ^^

Reply:
deleted